tests/server/test_endpoints.py

```diff
@@ -293,3 +295,73 @@ def test_dynamic_add_and_remove_endpoint(engine_mock):
     )
     assert response.status_code == 200
     assert 404 == client.post("/predict", json=dict(sequences="asdf")).status_code
+
+
+def test_pytorch_num_threads():
+    try:
+        import torch
```

Comment:
I recommend using `torch = pytest.importorskip("torch")` here ([docs](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip-ref)). That way it's clear the test is skipped rather than successfully passed

Reply:
OMG i could not figure out how to do this, thanks!!

Reply:
Added! beautiful 😁